enhancements/cluster-logging/hypershift-audit-logs.md

```diff
@@ -0,0 +1,219 @@
+---
+title: hypershift-audit-logs
+authors:
+  - aconway@redhat.com
+reviewers:
+  - TBD
+approvers:
+  - TBD
+api-approvers:
+  - TBD
+creation-date: 2023-03-31
+tracking-link:
+  - TBD
+see-also:
+  - /enhancements/cluster-logging/api-audit-log-policy.md
+---
+
+# Hypershift Audit Logs
+
+## Summary
+
+In a self contained cluster, users can forward application, infrastructure and audit logs.
+In a hypershift cluster, API server audit logs do not exist on the guest cluster,
+they are generated in Hypershift Control Plane (HCP) pods on the management cluster.
+
+This proposal deploys the existing cluster logging operator (CLO) on the management cluster.
+It will forward HCP audit logs according to a policy and destination configured on the guest cluster,
+and independently forward audit logs with the policy/ destination required by hypershift SD.
+
+**NOTE**: This proposal depends on [api-audit-log-policy](/api-audit-log-policy.md)
+
+## Motivation
+
+- Allow users and SREs with conflicting requirements to get the logs they need.
+- Forward API server logs from the management cluster, as they are not available on the guest cluster.
+- Replace the SD Splunk forwarder with the CLO which is more flexible and feature-rich.
+- Centralize log forwarding in the logging project, give hypershift team control of their logging needs.
```

Comment:
I don't see centralized forwarding necessarily as a motivation.  It's an optimization and an implementation detail.